scripts/experience_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "Installing Chrome web driver..."
+source <(curl -sSL "${DEVX_SKIT_ASSETS_GIT_URL_RAW:-https://github.com/IBM/devex-skit-assets/raw/v1.0.0}/scripts/install_chrome.sh")
```

Comment:
Can you use the latest release of the devex-skit-assets repo? This reminds me we need to update our skits as well.

Reply:
I don't use this at all.  It is here for you.  What URL would you like?